### ISSUE TITLE
Tweaks to automated review assignments

### ISSUE THREAD

**Issue by brian-rose** [ProjectPythia/projectpythia.github.io#158]
Two issues I've noticed since we starting automating reviewer assignments based on CODEOWNERS:

1. Two individuals from the @ProjectPythia/infrastructure team get selected via round-robin for reviews. That's what we want. But the review request addressed generically to @ProjectPythia/infrastructure remains. Examples are #153, #150. I was expecting that the generic request would get removed after the two individuals got tagged (which is how it's been working on the Foundations repo). In practice this means that when I look up my list of review requests to see what I'm supposed to work on, my list includes pretty much every open PR even when I'm not one of the individuals who's been tagged, and so the list is no longer very helpful.
2. Seems that things are getting tagged for review by @ProjectPythia/infrastructure that would be more appropriate for review by @ProjectPythia/education. This includes PRs specifically tagged as "content" like #150, and community-contributed gallery content like #153 

Maybe this is fodder for next week's IWG discussion.

**Comment by dopplershift** [comment]
1. I'll note that both of those are PRs from branches that are in this repository. I'm not sure if that has something to do with why the infrastructure team isn't being removed.
2. This is a direct consequence of [`CODEOWNERS`](https://github.com/ProjectPythia/projectpythia.github.io/blob/main/.github/CODEOWNERS) currently listing infrastructure as owners of everything and edu only being added as owners of `/content/`. Not sure what the right breakdown is, but definitely room for some more precision.

**Comment by jukent** [comment]
I'd like to have the appropriate team be assigned for review only after the "infrastructure", "content", or "community" labels are added. Currently the process happens as soon as a PR is created, regardless of label, depending on what files are edited.

**Comment by brian-rose** [comment]
Related, is there a way to set new PRs to default to "Draft" mode so that the author has some control over when the review requests are first generated?

Basically I'd like us to avoid sending out too many false review request notifications so that we don't become numb to them and start ignoring them.

**Comment by dopplershift** [comment]
I don't believe you can make it default, but when creating a new PR there's a small drop down next to the green Create Pull Request button the lets you change to Create Draft PR.

**Comment by brian-rose** [comment]
Point 2 above was fixed with #165. Just wondering if we should close this issue, or is there another fix that's possible to address point 1 (removing the team tag)? Seems that the problem is limited to cases where the PR is from a branch of this repo rather than someone's fork. This will happen whenever a PR is opened automatically from our "submit a link" action, for example.

**Comment by dopplershift** [comment]
I think you might have missed https://github.com/ProjectPythia/projectpythia.github.io/pull/165#issuecomment-926056637 in #165. The team sticking around issue was a consequence of having *required* reviews from codeowners enabled, which I have now disabled. I *think* everything should be working now?

**Comment by brian-rose** [comment]
Ahh thanks! I did miss that. And I haven't seen any new evidence of weirdness in the reviewer assignments. So I will close this now.
